jdbc/src/test/java/io/kestra/jdbc/JdbcTestUtils.java

```diff
@@ -38,14 +38,10 @@ public void drop() {
                 .getTables()
                 .stream()
                 .filter(throwPredicate(table -> (table.getSchema().getName().equals(dataSource.getConnection().getCatalog())) ||
-                    configuration.dialect() == SQLDialect.POSTGRES
+                    table.getSchema().getName().equals("public") // for Postgres
```

Comment:
I'm not sure for that, why the configuration dialect thing was put in first place ?

Reply:
No idea but for Postgres the schema is public but the configured catalog is kestra.
Anyway, it's for testing only so if all tests pass what's the issue?